nav-app/src/app/datatable/data-table.component.html

```diff
@@ -116,6 +116,23 @@
                                             [(ngModel)]="viewModel.description">
                                 </textarea>
                             </mat-form-field>
+
+                            <!-- layer domain (uneditable) -->
+                            <mat-form-field>
+                                <input matInput readonly
+                                    type="text"
+                                    placeholder="domain"
+                                    [(ngModel)]="dataService.getDomain(viewModel.domainID).name" />
+                            </mat-form-field>
+
+                            <!-- layer version (uneditable) -->
+                            <mat-form-field>
+                                <input matInput readonly
+                                    type="text"
+                                    placeholder="version"
+                                    [(ngModel)]="dataService.getDomain(viewModel.domainID).version" />
+                            </mat-form-field>
```

Comment:
These fields, though readonly, visually appear as if they're editable. Therefore it might be worth adding the `disabled` attribute, which should change them to be visually distinct while still following the overall pattern.

Example of what disabled looks like (note: it doesn't add the word `(disabled)` to the label, that's manually added in the example): 
![image](https://user-images.githubusercontent.com/28868888/100237959-9e28a700-2efd-11eb-8636-c700c2eb31a7.png)